config/crd/bases/leaderworkerset.x-k8s.io_leaderworkersets.yaml

```diff
@@ -8184,6 +8184,16 @@ spec:
                         - containers
                         type: object
                     type: object
+                  resizePolicy:
+                    default: None
+                    description: |-
+                      ResizePolicy defines how to handle group size updates:
+                      - None: indicates the `spec.leaderWorkerTemplate.size` is immutable.
+                      - Recreate: indicates recreating the group Pods to reshape the size.
+                    enum:
+                    - None
+                    - Recreate
+                    type: string
                   restartPolicy:
```

Comment:
Do the CRDs in the charts also need to be modified :-)

ref: https://github.com/kubernetes-sigs/lws/blob/main/charts/lws/templates/crds/leaderworkerset.x-k8s.io_leaderworkersets.yaml